UITests/Global.swift

```diff
@@ -78,10 +78,19 @@ class SimplePageServer {
         }
 
         webServer.addHandlerForMethod("GET", path: "/", requestClass: GCDWebServerRequest.self) { (request) -> GCDWebServerResponse! in
-            let page = (request.query["page"] as! String).toInt()!
             var pageDataPath = NSBundle(forClass: self).pathForResource("baseFile", ofType: "html")!
             var pageData = NSString(contentsOfFile: pageDataPath, encoding: NSUTF8StringEncoding, error: nil)!
+
+            if let title = request.query["title"] as? String {
+                // If the request doesn't want a page title, replace the title tags with <hiddenTitle>.
+                if title == "hidden" {
+                    pageData = pageData.stringByReplacingOccurrencesOfString("title>", withString: "hiddenTitle>")
```

Comment:
This is...terrible. Can you just create a separate page for this test? Trying to make the page work for every test will just turn it into an unmaintainable mess.